### PR TITLE
Pulpcore packages bump

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: packages/python-pulp-container/python-pulp-container.spec [hunk 1]
```diff
@@ -28,14 +28,12 @@ Summary:        %{summary}
...
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-ecdsa >= 0.15
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore < 3.20
-Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore >= 3.17.0
+Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore >= 3.18.0
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pyjwkest >= 1.4.0
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-pyjwkest >= 1.5
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pyjwt >= 1.7.1
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-pyjwt >= 1.8
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-setuptools
-Requires:       %{?scl_prefix}python%{python3_pkgversion}-url-normalize >= 1.4.2
```
Comment (pcreech) [inline packages/python-pulp-container/python-pulp-container.spec:37]: I'm assuming this is replaced by the dependency on url-normalize by the pulpcore package?

**Review by zjhuntin — APPROVED** [review]

**Comment by pcreech** [comment]
also looks like repoclosure blew up on el9 pkgs.  Is that expected at this point?

**Comment by Odilhao** [comment]
I didn't sign any package, so I'm not looking at repoclosure right now

File: packages/python-pulp-container/python-pulp-container.spec (hunk 1 above)
Comment (Odilhao) [reply packages/python-pulp-container/python-pulp-container.spec:37]: Looks like, it was removed from 2.11 

https://github.com/pulp/pulp_container/blob/74dffa939893e5ef5600d3da85bf01160f15a2e4/requirements.txt#L1

**Review by pcreech — APPROVED** [review]
Merge away when ready!

**Pull request merged**